### PR TITLE
docs: details for dynamic mode and globalstats

### PR DESCRIPTION
**Pull request by chrysan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -580,15 +610,29 @@ SHOW STATS_TOPN [ShowLikeOrWhere];
...
 可以通过执行 `DROP STATS` 语句来删除统计信息。
 
-语法如下：
-
 {{< copyable "sql" >}}
 
 ```sql
 DROP STATS TableName;
 ```
+该语句会删除 TableName 中所有的统计信息，包括分区动态裁剪模式下生成的 GlobalStats。
```
Comment (xuyifangreeneyes) [inline statistics.md:618]: 如果是分区表，包括所有分区的统计信息和分区动态裁剪模式下生成的 GlobalStats。感觉这样更清晰一些？

File: statistics.md [hunk 2]
```diff
@@ -285,6 +285,20 @@ ANALYZE TABLE TableName INDEX [IndexNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DE
     ANALYZE TABLE TableName PARTITION PartitionNameList [COLUMNS ColumnNameList|PREDICATE COLUMNS|ALL COLUMNS] [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
     ```
 
+##### 动态裁剪模式下的分区表统计信息
+
+分区表在开启[动态裁剪模式](/partitioned-table.md#动态裁剪模式) 的情况下，需要表级别的汇总的统计信息，以下称 GlobalStats。 目前 GlobalStats 由分区统计信息合并汇总得到。在动态裁剪模式开启的情况下，任一分区上的统计信息更新都会触发 GlobalStats 的更新。
```
Comment (shichun-0415) [inline statistics.md:290]:
```suggestion
分区表在开启[动态裁剪模式](/partitioned-table.md#动态裁剪模式)的情况下，需要表级别的汇总的统计信息，以下称 GlobalStats。 目前 GlobalStats 由分区统计信息合并汇总得到。在动态裁剪模式开启的情况下，任一分区上的统计信息更新都会触发 GlobalStats 的更新。
```

File: statistics.md [hunk 3]
```diff
@@ -401,6 +415,22 @@ TiDB 支持持久化的配置项包括：
...
+- ANALYZE TABLE ... PARTITION ... WITH ... 所指定的分区配置只持久化到分区级别，不会影响表级别的持久化配置
+- 当 ANALYZE 语句指定了配置，且同时存在持久化配置时，按照 语句 > 分区 > 表 的优先级继承和重写配置信息
+
+分区表在[动态裁剪模式](/partitioned-table.md#动态裁剪模式)下 ANALYZE 时，配置持久化遵守：
+- ANALYZE TABLE 时只持久化表级别的配置
+- 分区的统计信息会继承使用表级别的持久化配置
+- GlobalStats 会使用表级别的持久化配置
+- ANALYZE TABLE ... PARTITION ... WITH ... 所指定的分区配置会被忽略，且不会被持久化
+
+
```
Comment (shichun-0415) [inline statistics.md:433]:
```suggestion

```

File: statistics.md [hunk 4]
```diff
@@ -580,15 +610,29 @@ SHOW STATS_TOPN [ShowLikeOrWhere];
...
+该语句只删除 PartitionNameList 中对应的分区的统计信息。
+
+{{< copyable "sql" >}}
+
+```sql
+DROP STATS TableName GLOBAL;
+```
+
+该语句只删除该表在分区动态裁剪模式下生成的 GlobalStats。
```
Comment (shichun-0415) [inline statistics.md:635]:
```suggestion
```

File: statistics.md (hunk 1 above)
Comment (chrysan) [reply statistics.md]: Done

File: statistics.md [hunk 5]
```diff
@@ -285,6 +285,19 @@ ANALYZE TABLE TableName INDEX [IndexNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DE
...
+分区表在开启[动态裁剪模式](/partitioned-table.md#动态裁剪模式)的情况下，需要表级别的汇总的统计信息，以下称 GlobalStats。 目前 GlobalStats 由分区统计信息合并汇总得到。在动态裁剪模式开启的情况下，任一分区上的统计信息更新都会触发 GlobalStats 的更新。
+
+> **注意：**
+>
+> - 当触发 GlobalStats 更新时：
+
+>     - 若某些分区上缺少统计信息（比如新增的未 analyze 过的分区），会停止生成 GloalStats， 并通过 warning 信息提示用户缺少分区的统计信息。
+>     - 若某些列的统计信息合并过程中，缺少某些分区在该列上的统计信息（在不同分区上 analyze 时指定了不同的列），会停止生成 GloalStats，并通过 warning 信息提示用户缺少列在分区上的统计信息。
+
+> - 在动态裁剪模式开启的情况下，分区和表的 ANALYZE 配置需要保持一致，因此 ANALYZE TABLE TableName PARTITION PartitionNameList 命令后指定的 COLUMNS 配置和 WITH 后指定的 OPTIONS 配置将被忽略，并会通过 warning 信息提示用户。
```
Comment (shichun-0415) [inline statistics.md:299]:
```suggestion
> - 当触发 GlobalStats 更新时：
>
>     - 若某些分区上缺少统计信息（比如新增的未 analyze 过的分区），会停止生成 GloalStats， 并通过 warning 信息提示用户缺少分区的统计信息。
>     - 若某些列的统计信息合并过程中，缺少某些分区在该列上的统计信息（在不同分区上 analyze 时指定了不同的列），会停止生成 GloalStats，并通过 warning 信息提示用户缺少列在分区上的统计信息。
>
> - 在动态裁剪模式开启的情况下，分区和表的 ANALYZE 配置需要保持一致，因此 ANALYZE TABLE TableName PARTITION PartitionNameList 命令后指定的 COLUMNS 配置和 WITH 后指定的 OPTIONS 配置将被忽略，并会通过 warning 信息提示用户。
```

File: statistics.md [hunk 6]
```diff
@@ -401,6 +414,22 @@ TiDB 支持持久化的配置项包括：
...
+- 分区的统计信息会继承使用表级别的持久化配置
+- ANALYZE TABLE ... PARTITION ... WITH ... 所指定的分区配置只持久化到分区级别，不会影响表级别的持久化配置
+- 当 ANALYZE 语句指定了配置，且同时存在持久化配置时，按照 语句 > 分区 > 表 的优先级继承和重写配置信息
+
+分区表在[动态裁剪模式](/partitioned-table.md#动态裁剪模式)下 ANALYZE 时，配置持久化遵守：
+
+- ANALYZE TABLE 时只持久化表级别的配置
+- 分区的统计信息会继承使用表级别的持久化配置
+- GlobalStats 会使用表级别的持久化配置
+- ANALYZE TABLE ... PARTITION ... WITH ... 所指定的分区配置会被忽略，且不会被持久化
```
Comment (shichun-0415) [inline statistics.md:432]:
```suggestion
- ANALYZE TABLE ... PARTITION ... WITH ... 所指定的分区配置会被忽略，且不会被持久化

```

File: statistics.md [hunk 7]
```diff
@@ -285,6 +285,19 @@ ANALYZE TABLE TableName INDEX [IndexNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DE
     ANALYZE TABLE TableName PARTITION PartitionNameList [COLUMNS ColumnNameList|PREDICATE COLUMNS|ALL COLUMNS] [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
     ```
 
+##### 动态裁剪模式下的分区表统计信息
+
+分区表在开启[动态裁剪模式](/partitioned-table.md#动态裁剪模式)的情况下，需要表级别的汇总的统计信息，以下称 GlobalStats。 目前 GlobalStats 由分区统计信息合并汇总得到。在动态裁剪模式开启的情况下，任一分区上的统计信息更新都会触发 GlobalStats 的更新。
```
Comment (qiancai) [inline statistics.md:290]:
```suggestion
分区表在开启[动态裁剪模式](/partitioned-table.md#动态裁剪模式)的情况下，TiDB 将收集表级别的汇总统计信息，以下称 GlobalStats。 目前 GlobalStats 由分区统计信息合并汇总得到。在动态裁剪模式开启的情况下，任一分区上的统计信息更新都会触发 GlobalStats 的更新。
```

File: statistics.md [hunk 8]
```diff
@@ -401,6 +414,23 @@ TiDB 支持持久化的配置项包括：
 >
 > 当再次开启 `ANALYZE` 配置持久化功能时，如果之前记录的持久化配置项已经不适用当前的数据，请手动执行 `ANALYZE` 语句并指定新的持久化配置项。
 
+
+#### 分区表的 ANALYZE 配置持久化功能
+
+分区表在静态裁剪模式下 ANALYZE 时，配置持久化遵守：
```
Comment (qiancai) [inline statistics.md:420]:
```suggestion
在静态裁剪模式下 ANALYZE 分区表时，配置持久化遵守：
```

File: statistics.md [hunk 9]
```diff
@@ -401,6 +414,23 @@ TiDB 支持持久化的配置项包括：
...
+
+- ANALYZE TABLE 时会持久化表级别的配置和实际被 ANALYZE 的所有分区的配置
+- 分区的统计信息会继承使用表级别的持久化配置
+- ANALYZE TABLE ... PARTITION ... WITH ... 所指定的分区配置只持久化到分区级别，不会影响表级别的持久化配置
+- 当 ANALYZE 语句指定了配置，且同时存在持久化配置时，按照 语句 > 分区 > 表 的优先级继承和重写配置信息
+
+分区表在[动态裁剪模式](/partitioned-table.md#动态裁剪模式)下 ANALYZE 时，配置持久化遵守：
```
Comment (qiancai) [inline statistics.md:427]:
```suggestion
在[动态裁剪模式](/partitioned-table.md#动态裁剪模式)下 ANALYZE 分区表时，配置持久化遵守：
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: statistics.md [hunk 10]
```diff
@@ -401,6 +414,23 @@ TiDB 支持持久化的配置项包括：
 >
 > 当再次开启 `ANALYZE` 配置持久化功能时，如果之前记录的持久化配置项已经不适用当前的数据，请手动执行 `ANALYZE` 语句并指定新的持久化配置项。
 
+
```
Comment (shichun-0415) [inline statistics.md:417]:
```suggestion
```

**Pull request merged**